src/pages/monetization/app-review-process/index.mdx

```diff
@@ -8,9 +8,9 @@ desc: "If you're into building LiveChat integrations, but don't know how to go a
 
 # Introduction
 
-The goal of the <a href="https://www.livechat.com/marketplace/" target="_blank" rel="noopener noreferrer">**LiveChat Marketplace**</a> is to provide trusted space for high quality LiveChat apps, plugins, and integrations.
+The goal of the <a href="https://www.livechat.com/marketplace/" target="_blank" rel="noopener noreferrer">**LiveChat Marketplace**</a> is to provide trusted space for high quality LiveChat and HelpDesk apps, plugins, and integrations.
```

Comment:
AFAIK, there'll be a separate Marketplace for HelpDesk apps

Reply:
For now, Sambor told me that - pomysł jest taki aby w każdym produkcie można było wyszukać apki z innych produktów. Domyślnie będzie się wyświetlać lista aplikacji dla danego produktu, czyli jak np. wejdziemy w HD to wyświetla się apki dla HD, ale w filtrach będzie można wyklikać jeszcze np. apki dla LC - so generally I am not sure how to put it now...especially that we still do not have the Marketplace for HD 🤔 ? For now, should I change it into `our Marketplace`?